### ISSUE TITLE
NTR: Mandelic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8613]
New term request Mandelic acid high in urine (segal_030513182601):

**Comment by pnrobinson** [comment]
Elevated urinary carboxylic acid HP:0040156
Mandelic acid is a monocarboxylic acid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000999
- label: Elevated urinary mandelic acid
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: 
- def: The amount of mandelic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8613
- synonyms:
